css-positioning/fixed.css

```diff
@@ -0,0 +1,11 @@
+html {
+  font-size: 22px;
+}
+
+.page-title {
+  font-size: 3rem;
+  text-align: center;
+  position: fixed;
+  left: 50%;
+  margin-left: -190px;
```

Comment:
😱 😱 😱 😱 

Negative margins are bad news. You can likely remove these, and use `text-align: center` and `width: 100%` to achieve the same goal.